tests/src/TestBCTournament.cpp

```diff
@@ -111,4 +145,22 @@ void TestBCTournament::LeaveTournament()
 	TestResult tr;
 	m_bc->getTournamentService()->leaveTournament(_leaderboardId, &tr);
 	tr.run(m_bc);
+}
+
+int32_t TestBCTournament::JoinDivision()
+{
+	TestResult tr;
+	m_bc->getTournamentService()->joinDivision(_divSetId, _tournamentCode, 100, &tr);
+	tr.run(m_bc);
+
+	m_bc->getTournamentService()->getDivisionInfo(_divSetId, &tr);
```

Comment:
GetDivisionInfo test already exists. This serves no purpose, noone uses the returned value.